client/verta/verta/_internal_utils/_utils.py

```diff
@@ -528,7 +528,12 @@ def raise_for_http_error(response):
         curr_time = timestamp_to_str(now(), utc=True)
         time_str = " at {} UTC".format(curr_time)
 
-        reason = six.ensure_str(response.text.strip())
+        try:
+            reason = six.ensure_str(body_to_json(response)['message'])
```

Comment:
Need to call ensure str, otherwise would get `'ascii' codec can't encode character u'\u201c' in position 18: ordinal not in range(128)` error.

Reply:
Out of curiosity, did you test this in Python 2? (Could you try it out in both 2 and 3, if you can?)

Reply:
Just tried for both. LGTM.